php-frontend/src/main/java/org/sonar/php/tree/impl/expression/AnonymousClassTreeImpl.java

```diff
@@ -92,6 +101,12 @@ public List<AttributeGroupTree> attributeGroups() {
     return attributeGroups;
   }
 
+  @Nullable
+  @Override
+  public SyntaxToken readonlyToken() {
+    return readonly;
+  }
```

Comment:
I understand why you implemented this getter, but I think a getter like `boolean isReadOnly` will be more useful. So far we will not need to access the token explicitly.